tools/scan-build-py/README.md

```diff
@@ -41,6 +41,31 @@ goes like this:
 Use `--help` to know more about the commands.
 
 
+How to use the experimental Cross Translation Unit analysis
+-----------------------------------------------------------
+
+To run the CTU analysis, a compilation database file has to be created:
+
+    $ intercept-build <your build command>
+
+To run the Clang static analyzer against a compilation database
+with CTU analysis enabled, execute:
+    
+    $ analyze-build --ctu
+
```

Comment:
Please add this sentence: 

For CTU analysis an additional (function-definition) collection-phase is needed.